src/lwc/geListView/geListView.js

```diff
@@ -374,64 +383,96 @@ export default class geListView extends LightningElement {
         return options;
     }
 
-    /*******************************************************************************
-    * @description Method builds a list of options used to populate the available
-    * fields in the utilDualListbox component. utilDualListbox is used in the list
-    * settings modal.
-    *
-    * @param {list} headerFieldApiNames: List of fields from the object describe info.
-    */
-    setDatatableColumns(headerFieldApiNames) {
+    /************************************************************************************************************
+     * @description Convert all reference type columns so that column values can be dynamically assigned to each
+     * column in the lightning data table
+     *
+     * @param {array} fieldApiNames: List of field api names to check if reference conversation is needed
+     * @return array
+     */
+    buildNameFieldColumns(fieldApiNames) {
         this.columnEntriesByName = {};
-        this.columns = [];
+        let _columns = [];
+
+        for (let i = 0; i < fieldApiNames.length; i++) {
```

Comment:
@vmartinov-sforg We wouldn't be able to use `const` in this case since we're setting the `fieldApiName` variable further down on line 419, but I can use the name syntax you recommended and just use the `let` assignment instead of `const`. Just pushed that up.